config/docker-entrypoint.sh

```diff
@@ -1,32 +1,38 @@
 #!/bin/sh
 set -e
 
-case "$1" in
-  prometheus)
-    shift
+waitFn() { :; }
 
-    cat <<-EOF >helm.yaml
-		apiVersion: konjure.carbonrelay.com/v1beta1
-		kind: HelmGenerator
-		metadata:
-		  name: prometheus
-		releaseName: optimize-${NAMESPACE}-prometheus
-		releaseNamespace: ${NAMESPACE}
-		chart: ../prometheus
-		EOF
-
-    export HELM_CONFIG=$(cat helm.yaml | base64 -w0)
-
-    waitFn() {
-      # Wait on {{ releaseName }}-server
-      kubectl wait --for condition=Available=true --namespace "${NAMESPACE}" --timeout 5m deployment.apps "optimize-${NAMESPACE}-prometheus-server"
-    }
-  ;;
-  *)
-    waitFn() { :; }
-  ;;
-esac
+while [ "$#" != "0" ] ; do
+  case "$1" in
+    prometheus)
+      shift
+
+      cat <<-EOF >helm.yaml
+			apiVersion: konjure.carbonrelay.com/v1beta1
+			kind: HelmGenerator
+			metadata:
+			  name: prometheus
+			releaseName: optimize-${NAMESPACE}-prometheus
+			releaseNamespace: ${NAMESPACE}
+			chart: ../prometheus
+			EOF
+
+      export HELM_CONFIG=$(cat helm.yaml | base64 -w0)
```

Comment:
I don't think I noticed this before, but `-w0` isn't something you can do with the Busy Box `base64`.

Reply:
Yeah, that's the reason for installing coreutils. I had a similar confused look because I hadn't noticed it either in testing. I put some details in the commit message for https://github.com/thestormforge/optimize-controller/pull/438/commits/80da0f8fe0616d353212851f6de7e2a26e0dd8ed

Reply:
I'd think with quotes you'd be fine, even in ash.
```
export HELM_CONFIG="$(base64 helm.yaml)"
```

Reply:
Or keep the pipe into `base64` and eliminate the file system write
```
export HELM_CONFIG="$(cat <<-EOF | base64
		apiVersion...
		...
		chart: ../prometheus
		EOF)"
```